src/pages/top-up/Swap.tsx

```diff
@@ -44,6 +44,11 @@ export function Swap({ header }: Props): ReactElement {
   const navigate = useNavigate()
   const { enqueueSnackbar } = useSnackbar()
 
+  useEffect(() => {
+    // eslint-disable-next-line no-console
+    getBzzPriceAsDai().then(setPrice).catch(console.error)
+  }, [])
```

Comment:
Hmm that means that the price is fetched only once per the lifetime of the app right? Shouldn't there be some auto refresh system for this?